packages/paypal-commerce-utils/src/paypal-commerce-types.ts

```diff
@@ -26,6 +26,7 @@ export interface PayPalCommerceInitializationData {
     isDeveloperModeApplicable?: boolean;
     intent?: PayPalCommerceIntent;
     isAcceleratedCheckoutEnabled?: boolean; // PayPal Fastlane related
+    isEnableFastlaneShippingOptionAutoSelect?: boolean; // PayPal Fastlane related
```

Comment:
Can we update name of the option?
```suggestion
    isFastlaneShippingOptionAutoSelectEnabled?: boolean; // PayPal Fastlane related
```

Reply:
fixed